doc/source/explanation/index.rst

```diff
@@ -22,6 +22,6 @@ There are also support materials for
 
-So, what is Girder 4? Built on all these software pieces, you can think of it as a *methodology, including best practices and composable utilities, for building data-focused web applications using Django, Django REST Framework, PostgreSQL, and Celery.* This selection of libraries, configurations, and managed service offerings is the product of our team’s combined decades of web service development and maintenance experience. We wrote the libraries that are a part of Girder 4 to capture our evolving best practices and empower ourselves to work both *efficiently* and *effectively* on customer projects. The software is all open source, so it is free and available to use in your own projects. But remember that underneath the powerful parts we have contributed, Girder 4 projects are Django projects, and that we spend much of our thinking on understanding the domain and modeling it appropriately, so that the web applications we create enable our customers and collaborators to achieve deeper insight and power with their data.
+So, what is Resonant? Built on all these software pieces, you can think of it as a *methodology, including best practices and composable utilities, for building data-focused web applications using Django, Django REST Framework, PostgreSQL, and Celery.* This selection of libraries, configurations, and managed service offerings is the product of our team’s combined decades of web service development and maintenance experience. We wrote the libraries that are a part of Resonant to capture our evolving best practices and empower ourselves to work both *efficiently* and *effectively* on customer projects. The software is all open source, so it is free and available to use in your own projects. But remember that underneath the powerful parts we have contributed, Resonant projects are Django projects, and that we spend much of our thinking on understanding the domain and modeling it appropriately, so that the web applications we create enable our customers and collaborators to achieve deeper insight and power with their data.
 
-And remember that Girder is still Girder! We are able to do the same rich application building and problem solving you may be used to with Girder 3, but we have learned many lessons and improved our process, knowledge, and understanding in moving to Girder 4. And of course, we are still supporting Girder 3 projects that already exist—we are simply building *new* projects with the enhanced insight and understanding we have captured in the Girder 4 family of tools.
+We are able to do the same rich application building and problem solving you may be used to with Resonant, but we have learned many lessons and improved our process, knowledge, and understanding in moving to Resonant. And of course, we are still supporting Girder  projects that already exist—we are simply building *new* projects with the enhanced insight and understanding we have captured in the Resonant family of tools.
```

Comment:
> And of course, we are still supporting Girder  projects that already exist—we are simply building *new* projects with the enhanced insight and understanding we have captured in the Resonant family of tools.

I don't know if this is still accurate. Are we expecting new Girder 3 projects? If so should this sentence just be removed?

Reply:
I think we can cut this entire line. It just muddies the water.